std/logger/package.d

```diff
@@ -54,6 +54,7 @@ $(UL
     $(LI `trace`)
     $(LI `info`)
     $(LI `warning`)
+    #(LI `error`)
```

Comment:
@HuskyNator the syntax for DDoc macros is `$(MACRO_NAME ARGS)`, see: https://dlang.org/spec/ddoc#macros

```suggestion
    $(LI `error`)
```

Reply:
@thewilsonator please check the doc tester before merging:

![image](https://github.com/dlang/phobos/assets/3475150/8d7a41cd-e3bd-4cf4-8a5c-aaf32fe4ffa2)

https://dtest.dlang.io/artifact/website-fd7e1f50f4d2329f24313ea92bfcbd91af962cdb-57c7f9baeef70e48134208b01ac92871/web/phobos-prerelease/std_logger.html

Reply:
I've opened a PR to fix this: https://github.com/dlang/phobos/pull/8847